common/perf-test-core/src/main/java/com/azure/perf/test/core/HttpPipelineTest.java

```
@@ -62,6 +63,21 @@ public Mono<Void> runAsync() {
         }
     }
 
+    @Override
+    public CompletableFuture<Void> runAsyncWithCompletableFuture() {
+        return null;
```

Let's throw here as `UnsupportedOperationException`